praktikumsplaner-frontend/frontend/src/api/ExcelService.ts

```diff
@@ -0,0 +1,34 @@
+import FetchUtils from "@/api/FetchUtils";
+import { API_BASE, NWK_BASE } from "@/Constants";
+
+export default {
+    uploadExcelFile(excelDatei: File): Promise<void> {
+        // File Reader encodes as Base64
+        return this.readString(excelDatei).then((base64string: string) => {
+            return fetch(
+                `${API_BASE}${NWK_BASE}/import`,
+                // Base64 String starts after the comma
+                FetchUtils.getPOSTConfig(base64string.split(",")[1])
+            )
+                .then((response) => {
+                    FetchUtils.defaultResponseHandler(response);
+                })
+                .catch((err) => {
+                    FetchUtils.defaultResponseHandler(err);
+                });
+        });
+    },
+    readString(excelDatei: File): Promise<string> {
+        return new Promise((resolve, reject) => {
+            const reader = new FileReader();
+            reader.onloadend = () => {
+                if (!(typeof reader.result == "string")) {
+                    return Promise.reject();
+                }
+                resolve(reader.result);
+            };
+            reader.onerror = reject;
+            reader.readAsDataURL(excelDatei);
+        });
```

Comment:
das mit einem Test abzudecken wäre aus meiner Sicht etwas für den Goldstandard. Daher schlage ich vor das Thema in ein technisches Ticket zu gießen.

@mirrodi @AnHo314 wie seht ihr das?

Reply:
Ich glaube auch, dass man sich das erstmal sparen kann. Ein technischen Ticket ist ein guter Mittelweg